src/resources/postcss/base/forms/_text.pcss

```diff
@@ -24,27 +24,44 @@
 	}
 
 	.tribe-common-form-control-text__input {
-		@mixin desktop-body-2-bold;
-
-		border: 1px solid var(--color-border-default);
-		padding: 12px 18px;
+		@mixin desktop-body-1;
+		border: none;
+		border-bottom: 1px solid var(--color-border-default);
+		padding: 12px 20px 12px 0;
 		width: 100%;
 
+		@media (--viewport-medium) {
+			@mixin desktop-body-2;
+
+			border: none;
+			padding: 12px 18px;
+		}
+
 		&::placeholder {
 			color: var(--color-text-secondary);
 			font-weight: var(--font-weight-regular);
 		}
+
+		&:focus {
+			border-bottom-color: var(--color-border-active);
+		}
 	}
 
 	body#top & { /* Enfold Compatibility */
 
 		.tribe-common-form-control-text__input {
-			@mixin desktop-body-2-bold;
+			@mixin desktop-body-1;
 
-			border: 1px solid var(--color-border-default);
-			padding: 12px 18px;
+			border-bottom: 1px solid var(--color-border-default);
+			padding: 12px 20px 12px 0;
 			width: 100%;
 
+			@media (--viewport-medium) {
+				@mixin desktop-body-2;
+
+				padding: 12px 18px;
```

Comment:
let's use `--spacer` for these. also, i think these are different now, confirm with the desktop designs.

Reply:
@juanfra were you able to confirm with the designs? I think matt is using `--spacer-4` for top and bottom.

Reply:
He's using `--spacer-4` but in this way: https://cloudup.com/cwJOV8799UJ - That's not replicable on HTML, and we'll have to find a way to make it look like that without having 20px padding how he demonstrated on the sketch.

Reply:
Let's chat about this, we can replicate this on HTML.